packages/govuk-frontend/package.json

```diff
@@ -2,27 +2,27 @@
   "name": "govuk-frontend",
   "description": "GOV.UK Frontend contains the code you need to start building a user interface for government platforms and services.",
   "version": "4.6.0",
-  "main": "govuk/all.js",
-  "module": "govuk-esm/all.mjs",
+  "main": "dist/govuk/all.js",
+  "module": "dist/govuk-esm/all.mjs",
   "files": [
     "dist",
     "package.json",
     "README.md"
   ],
   "exports": {
     ".": {
-      "sass": "./govuk/all.scss",
-      "import": "./govuk-esm/all.mjs",
-      "require": "./govuk/all.js"
+      "sass": "./dist/govuk/all.scss",
+      "import": "./dist/govuk-esm/all.mjs",
+      "require": "./dist/govuk/all.js"
     },
-    "./govuk/": "./govuk/",
-    "./govuk-esm/": "./govuk-esm/",
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./src/*": "./src/*",
```

Comment:
**question** Given `src` is not part of `files` will that break anything for users that it's still listed as `exports`? I'm guessing it needs to be here for use in the review app?

Reply:
Good point. But no, this line assumes we're going to start publishing `src`

I'd keep it if we want to link source maps, declaration maps (and debuggers) to the original code
I'd remove it if we decide to exclude `src`

Reply:
I've gone for `"./*": "./*"` so we can access anything, src or dist

(But no magic remapping of paths)